src/elements/content-sidebar/activity-feed/activity-feed/ActivityItem.scss

```diff
@@ -4,9 +4,9 @@
     &.hasNewThreadedReplies {
         padding: 2px $bdl-grid-unit * 2;
 
+        // This brittle selector is here in order to only select the parent BaseComment and not the BaseComments which constitute its replies.
         & > div {
```

Comment:
Any way to make it less brittle ?

Reply:
Yes, actually. And I would love to do so. I think we need to restructure a bit so that the focus-implying border is on a wrapper for the BaseComment rather than on the BaseComment itself. Then we don't need to distinguish the first BaseComment from its replies.

Reply:
This focus border should only be on the outer BaseComment? In that case I agree, it shouldn't be part of BaseComment and should be factored out to a wrapper

Reply:
Heads up, I have not handled this in this PR. Would it be alright to break this out into a separate task?

Reply:
Yea I think that will work.